src/encoded/commands/generate_annotations.py

```diff
@@ -85,7 +85,7 @@ def human_single_annotation(r):
 
         # Adding gene synonyms to autocomplete
         if r['Synonyms'] is not None and r['Synonyms'] != '':
-            synonyms = [x.strip(' ') + species for x in r['Synonyms'].split(',')]
+            synonyms = [x.strip(' ').strip('\"') + species for x in r['Synonyms'].split(',')]
```

Comment:
I thought a made a comment on here earlier.  

This is not really a bug fix.  The slash-quote issue is not in the original tsv so we're adding it in the generate annotations process.  This only undoes the original issue, which is probably the quoted lists in the original tsv being decoded here.

https://github.com/ENCODE-DCC/encoded/blob/dev/src/encoded/commands/generate_annotations.py#L190

Did you look into fixing this issue at the source, get_rows_from_file or human_annotations?

Reply:
@caseylitton I missed your comment about "address the root cause." Excellent point. I am looking into now.

Reply:
In this file (generate_annotations.py), line 191 and 192, a double quote-string is at the beginning and end of the Synonym (like- '"apple"'). So, what I did was remove them and added a comment to try to make that clear.

Reply:
actually I think this fix at this line IS correct because he has to parse strings that look like "A, B, C" into [A,B,C].  So he needs to strip the \"

However field 3, "Approved Name" ALSO has (sometimes extraneous \" in it.  So either every field has to be stripped or Name has to be stripped as well.

Reply:
Addressed. Stripped double quotes from all fields.